### DISCUSSION TITLE
add Styles API to TypographyStylesProvider

### DISCUSSION THREAD

**Discussion by ghifar1** [mantinedev/mantine#2563] [Feature requests]
hello, i cannot modify p tag in TypographyStylesProvider.
![image](https://user-images.githubusercontent.com/38382464/192511181-495ceab4-94b7-4286-8fff-12ce3d709bbe.png)
based on the picture above, i want to remove paragraph margin.
![image](https://user-images.githubusercontent.com/38382464/192511714-e0c4dbab-5b06-4755-b572-6e9f1a0a5ca4.png)

or do you have any other suggestions?, thanks.

**Comment by rtivital** [comment]
`<TypographyStylesProvider sx={{ '& p': { color: 'red' } }} />`

**Reply by ghifar1** [reply]
thank you
